spec/lib/audit/catalog_to_moab_spec.rb

```diff
@@ -71,7 +70,7 @@
       c2m.check_catalog_version
     end
 
-    it 'calls online_moab_found?(druid, storage_dir)' do
+    it 'calls online_moab_found?(druid, storage_location)' do
```

Comment:
looking through the code, i'm sort of guessing this parenthetical is cruft from a prior method signature.  or at least, i don't really understand what this parenthetical is communicating at present.  will leave a suggestion that gets rid of it, and you can squash it in (or not) if you'd like.

Reply:
Committed!